packages/runner-ct/src/app/BottomPane.jsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+
```

Comment:
Can we start to use `tsx` everywhere, at least for all new components? no reason not too

Reply:
It's not possible with runner-ct without configuration. Will be happy to migrate to PR in separate PR